src/Containers/ImageCarousel.tsx

```diff
@@ -20,6 +24,10 @@ export interface ImageCarouselProps
 export const ImageCarousel: React.FC<ImageCarouselProps> = ({
     imageData,
     onClick,
+    pointer,
+    icon,
```

Comment:
```suggestion
    icon = true,
```
Not sure why you don't just make this a source and let the person pass an Icon and if they don't pass an Icon then you don't render the Icon ;) This seperate icon is bad practice

Reply:
Ex: hoverIcon: source = "X"
hoverIcon -> Clear Name
: source -> Clear Type
"X" -> To not break the existing applciations using this component :)